packages/editor/src/extensions/link/helpers/pasteHandler.ts

```diff
@@ -20,32 +20,65 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import { Editor } from "@tiptap/core";
 import { MarkType } from "@tiptap/pm/model";
 import { Plugin, PluginKey } from "@tiptap/pm/state";
+import linkifyHtml from "linkify-html";
```

Comment:
Why aren't we using `linkifyjs`?

Reply:
I found `linkifyHtml` to be much simpler for this usecase

Reply:
Let's avoid adding a new dep. Either replace all usages of `linkify-js` with `linkify-html` or just use `linkify-js` everywhere.

Reply:
Makes sense